vm/systemSmartContracts/governance.go

```diff
@@ -910,6 +910,11 @@ func (g *governanceContract) EpochConfirmed(epoch uint32) {
 	log.Debug("governance contract", "enabled", g.flagEnabled.IsSet())
 }
 
+// SetNewGasCosts is called whenever a gas cost was changed
+func (g *governanceContract) SetNewGasCosts(gasCost vm.GasCost) {
+	g.gasCost = gasCost
```

Comment:
We need to protect `s.gasCost` with a mutex

Reply:
no need. there is no possibility to call the from different go routines

Reply:
protected all the way